x-pack/plugin/sql/src/main/antlr/SqlBase.g4

```diff
@@ -50,9 +50,9 @@ statement
         )*
         ')')?
         statement                                                                                         #debug
-    | SHOW TABLES (tableLike=likePattern | tableIdent=tableIdentifier)?                                   #showTables
-    | SHOW COLUMNS (FROM | IN) (tableLike=likePattern | tableIdent=tableIdentifier)                       #showColumns
-    | (DESCRIBE | DESC) (tableLike=likePattern | tableIdent=tableIdentifier)                              #showColumns
+    | SHOW TABLES (INCLUDE FROZEN)? (tableLike=likePattern | tableIdent=tableIdentifier)?                 #showTables
```

Comment:
This is the declarative approach for frozen which is useful when not using `index.include.frozen` (again `.` are used by drivers, `_` is used in rest).